extensions/wait/wait.go

```diff
@@ -0,0 +1,33 @@
+package wait
+
+import (
+	"github.com/rancher/shepherd/pkg/wait"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+func ResourceCreate(watchInterface watch.Interface) error {
```

Comment:
Can we get go doc comments here please

Reply:
Added some comments as per your suggestion.